graddft_qnn/main.py

```diff
@@ -1,3 +1,5 @@
+import os
```

Comment:
consider loading the  batch_size dynamically, this assume batch_size has to be >= 10, which may not be the case